doc/source/train/examples.yml

```diff
@@ -54,6 +54,26 @@ examples:
       - natural language processing
     contributor: community
     link: examples/intel_gaudi/bert
+  - title: "Fine-tune Llama-2-7b with Intel Gaudi"
```

Comment:
"Fine-tune Llama-2-7b and Llama-2-70b with Intel Gaudi"

Reply:
Updated!